tests/gene-details.spec.ts

```diff
@@ -7,11 +8,12 @@ test.describe('specific viewport block', () => {
   test('invalid gene results in a 404 redirect', async ({ page }) => {
     // go to invalid ENSG page
     await page.goto('/genes/ENSG00000000000');
+    await waitForSpinnerNotVisible(page);
```

Comment:
Waiting for loader to be removed reduces flakiness when page takes a long time to load